models/eos/ntp_global/eos_ntp_global.yml

```diff
@@ -0,0 +1,152 @@
+---
+module: eos_ntp_global
+short_description: Manages ntp resource module
+description: This module configures and manages the attributes of  ntp on Arista
+  EOS platforms.
+version_added: 3.1.0
+author: Gomathi Selvi Srinivasan (@GomathiselviS)
+notes:
+- Tested against Arista EOS 4.24.60M
+- This module works with connection C(network_cli). See the L(EOS Platform Options,eos_platform_options).
+options:
+    config:
+      description: A dictionary of ntp options
+      type: dict
+      suboptions:
+        authenticate:
+          description:
+          - Require authentication for NTP synchronization.
+          type: dict
+          suboptions:
+            enable:
+              description: Enable authentication for NTP synchronization.
+              type: bool
+            servers:
+              description: Authentication required only for incoming NTP server responses.
+              type: bool
+        authentication_key:
+          description:
+          - Define a key to use for authentication.
+          type: list
+          elements: dict
+          suboptions:
+            id:
+              description: key identifier.
+              type: int
+            algorithm:
+              description: hash algorithm,
+              type: str
+              choices: ["md5", "sha1"]
+            encryption:
+              description: key type
+              type: int
+              choices: [0, 1]
+            key:
+              description: Unobfuscated key string.
+              type: str
+              no_log: True
```

Comment:
Note: This will trigger validate-modules since `no_log` is not a part of doc schema (yet). You will have to remove this from the docstring once scaffolding is done.